src/testing/mocks/middleware/validity.ts

```diff
@@ -0,0 +1,39 @@
+import { create, invalidator } from '../../../core/vdom';
+import { DefaultMiddlewareResult } from '../../../core/interfaces';
+
+interface ValidityResult {
+	valid?: boolean;
+	message?: string;
+}
+
+export function createValidityMock() {
+	const values: { [key: string]: ValidityResult } = {};
+	let invalidate: () => void | undefined;
+
+	const factory = create({ invalidator });
+
+	const mockValidityFactory = factory(({ middleware: { invalidator } }) => {
+		invalidate = invalidator;
+		return {
+			get(key: string | number, value: string) {
+				value;
```

Comment:
is this meant to be here?

Reply:
Oops, no I meant to remove that. I originally added it because `value` is unused and I was quieting TypeScript. Renaming the argument to `_value` to quiet TypeScript instead.